backend/src/modules/auth/auth.service.ts

```diff
@@ -158,7 +167,7 @@ export class AuthService {
     const [accessToken, refreshToken] = await Promise.all([
       this.jwtService.signAsync(payload, {
         secret: `.${process.env.NODE_ENV}.${process.env.ACCESS_TOKEN_JWT_SECRET}`,
-        expiresIn: '10m',
+        expiresIn: '30d',
```

Comment:
access token should live small time
let's move it to env variable to do it configurable